collector.go

```diff
@@ -115,6 +116,9 @@ func (c *Collector) Collect(ctx context.Context) error {
 		}
 	}
 
+	sort.Sort(SortBundlesByVersion(bundles))
```

Comment:
Added sorting here because related unit tests were flapping for me. Root cause for this is randomization in `responses` ordering.

Reply:
I am fine with sorting here.